charts/sscs-tribunals-api/values.ccd.preview.template.yaml

```diff
@@ -27,7 +27,7 @@ java:
     IDAM_SSCS_SYSTEMUPDATE_USER: ${IDAM_SSCS_SYSTEMUPDATE_USER}
     IDAM_SSCS_SYSTEMUPDATE_PASSWORD: ${IDAM_SSCS_SYSTEMUPDATE_PASSWORD}
     CORE_CASE_DATA_JURISDICTION_ID: SSCS
-    CORE_CASE_DATA_CASE_TYPE_ID: ${CASE_TYPE}
+    CORE_CASE_DATA_CASE_TYPE_ID: Benefit-3860
```

Comment:
Make sure these are reverted before merge

Reply:
Reverted to case type

Reply:
values.ccd.preview.template.yaml file is already removed, please rebase with master